samples/matter/common/src/bridge/matter_bridged_device.cpp

```diff
@@ -62,17 +63,42 @@ CHIP_ERROR MatterBridgedDevice::HandleReadBridgedDeviceBasicInformation(Attribut
 	}
 }
 
-CHIP_ERROR MatterBridgedDevice::HandleReadDescriptor(AttributeId attributeId, uint8_t *buffer, uint16_t maxReadLength)
+CHIP_ERROR MatterBridgedDevice::HandleWriteIdentify(chip::AttributeId attributeId, void *data, size_t dataSize)
 {
 	switch (attributeId) {
-	case Clusters::Descriptor::Attributes::ClusterRevision::Id: {
-		uint16_t clusterRevision = GetDescriptorClusterRevision();
+	case Clusters::Identify::Attributes::IdentifyTime::Id:
+		if (data && dataSize == sizeof(mIdentifyTime)) {
+			mIdentifyTime = (*reinterpret_cast<decltype(mIdentifyTime) *>(data));
+			/* Externally stored attribute was updated, now we need to notify identify-server to
+			   leverage the Identify cluster implementation */
+			app::ConcreteAttributePath attributePath{ mEndpointId, Clusters::Identify::Id, attributeId };
+			MatterIdentifyClusterServerAttributeChangedCallback(attributePath);
```

Comment:
Actually, I'm not sure if the identify-server will be able to pick up the new value stored externally. I trust it will, but just giving you a heads up that it's worth checking as in the past attribute accessors that identify-server uses didn't work with external attributes.

Reply:
Yes, fortunately it works as expected, the identify-server eventually calls `emAfReadOrWriteAttribute()` and `emberAfAttributeWriteAccessCallback()`. The latter checks the attribute bitmask for `ATTRIBUTE_MASK_EXTERNAL_STORAGE` and calls `emberAfExternalAttributeWriteCallback`/`emberAfExternalAttributeWriteCallback` accordingly.

Reply:
ah, got it. Thanks for explanation!